### ISSUE TITLE
Does it play nice with Home Assistant?

### ISSUE THREAD

**Issue by mikepan-ovoenergy** [JensVanhooydonck/xknx-configuration-importer#13]
Does it play nice with Home Assistant?

**Comment by farmio** [comment]
Currently only with a detour via https://xknx.io/config_converter.html 

The config Schema it generates is deprecated in HA 2021.4

**Comment by JensVanhooydonck** [comment]
Indeed, still need to find some time to alter the importer to output the correct format for HA.

**Comment by mikepan-ovoenergy** [comment]
Maybe it's worth thinking for a Home Assistant add-on?

**Comment by farmio** [comment]
Wouldn't it have to be written in Python then?

**Comment by JensVanhooydonck** [comment]
I'v added a basic HomeAssistant Config layout as well. Will need to add more options in the future if i find the time. PR's are welcome (:
